### PR TITLE
docs(plugin): refactor traffic-split.md

### PR DESCRIPTION
**Pull request by lunarwhite**
### Description

update documents under `docs/zh/latest/plugins/` along with `docs/en/latest/plugins/`

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/traffic-split.md [hunk 1]
```diff
@@ -23,53 +28,54 @@ title: traffic-split
 
 ## 描述
 
-traffic-split 插件使用户可以逐步引导各个上游之间的流量百分比。
+`traffic-split` 插件可以通过配置 `match` 和 `weighted_upstreams` 属性，从而动态地将部分流量引导至各种上游服务。`match` 是用于分割流量的自定义规则，`weighted_upstreams` 是用于引导流量的一组上游服务。当一个请求被 `match` 属性匹配时，它将根据配置的 `weights` 属性被引导到上游服务。你也可以不使用 `match` 属性，而根据 `weighted_upstreams` 属性引导所有流量。
```
[the remark this answers is not in the document]
Comment (SylviaBABY) [reply docs/zh/latest/plugins/traffic-split.md:31]: 你可以使用该插件进行灰度发布和蓝绿发布的场景应用。

File: docs/zh/latest/plugins/traffic-split.md [hunk 2]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
 
 ## 描述
 
-traffic-split 插件使用户可以逐步引导各个上游之间的流量百分比。
+`traffic-split` 插件可以通过配置 `match` 和 `weighted_upstreams` 属性，从而动态地将部分流量引导至各种上游服务。该插件可应用于灰度发布和蓝绿发布的场景。
 
-注：由于加权循环算法（特别是在重置 wrr 状态时）的缺点，因此每个上游之间的比率可能不太准确。
+`match` 是用于引导流量的自定义规则，`weighted_upstreams` 是用于引导流量的一组上游服务。当一个请求被 `match` 属性匹配时，它将根据配置的 `weights` 属性被引导至上游服务。你也可以不使用 `match` 属性，而根据 `weighted_upstreams` 属性引导所有流量。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:35]:
```suggestion
`match` 属性是用于引导流量的自定义规则，`weighted_upstreams` 属性则用于引导流量的上游服务。当一个请求被 `match` 属性匹配时，它将根据配置的 `weights` 属性被引导至上游服务。你也可以不使用 `match` 属性，只根据 `weighted_upstreams` 属性来引导所有流量。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 3]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+
+:::note 注意
+
+由于使用了加权循环算法（特别是在重置 `wrr` 状态时），上游服务之间的流量比例可能不太准确。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:39]:
```suggestion
由于使用了加权循环算法（特别是在重置 `wrr` 状态时），因此在使用该插件时，可能会存在上游服务之间的流量比例不精准现象。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 4]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
-```json
-"weighted_upstreams": [
-    ......
-    {
-        "weight": 2
-    }
-]
-```
+| rules.match                    | array[object] | 否  |        |        | 匹配规则列表，默认为空且规则将被无条件执行。                                                                                                                                                                                                                                                                                                                                           |
+| rules.match.vars               | array[array]  | 否   |        |        | 由一个或多个 `{var, operator, val}` 元素组成的列表，例如：`{"arg_name", "==", "json"}`，表示当前请求参数 `name` 是 `json`。这里的 `var` 与 NGINX 内部自身变量命名是保持一致，所以也可以使用 `request_uri`、`host` 等；对于已支持的运算符，具体用法请看 [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list) 的 `operator-list` 部分。 |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:48]:
```suggestion
| rules.match.vars               | array[array]  | 否   |        |        | 由一个或多个 `{var, operator, val}` 元素组成的列表，例如：`{"arg_name", "==", "json"}`，表示当前请求参数 `name` 是 `json`。这里的 `var` 与 NGINX 内部自身变量命名是保持一致，所以也可以使用 `request_uri`、`host` 等；对于已支持的运算符，具体用法请参考 [lua-resty-expr](https://github.com/api7/lua-resty-expr#operator-list) 的 `operator-list` 部分。 |
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 5]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+| weighted_upstreams.upstream    | object | 否   |        |        | 上游配置信息。                                                                                                                                                                                                                                                                                                                                                          |
+| upstream.type                  | enum   | 否   |   roundrobin |  [roundrobin, chash]      | 流量引导机制的类型，`roundrobin` 支持权重的负载，`chash` 使用一致性哈希。                                                                                                                                                                                                                                                                                                                           |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:52]:
```suggestion
| upstream.type                  | enum   | 否   |   roundrobin |  [roundrobin, chash]      | 流量引导机制的类型；`roundrobin` 表示支持权重的负载，`chash` 表示使用一致性哈希。                                                                                                                                                                                                                                                                                                                           |
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 6]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+| upstream.key                   | string | 否   |      |    | 该属性仅当 `upstream.type` 是 `chash` 时有效。根据 `hash_on` 和 `key` 来查找对应的 Node `id`。更多信息请参考 [Upstream](../admin-api.md#upstream) 用法。                                                                                                                                                                                                                                    |
+| upstream.nodes                 | object | 否   |        |        | 哈希表，键是上游节点的 IP 地址与可选端口的组合，值是节点的权重。将 `weight` 设置为 `0` 表示一个请求永远不会被转发到该节点。                                                                                                                                                                                                             |
+| upstream.timeout               | object | 否   |  15     |        | 设置连接、发送消息、接收消息的超时时间（单位为秒）。                                                                                                                                                                                                                                                                                                                           |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:56]:
```suggestion
| upstream.timeout               | object | 否   |  15     |        | 设置连接、发送和接收消息的超时时间（单位为秒）。                                                                                                                                                                                                                                                                                                                           |
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 7]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
->注：1、通过 `upstream_id` 方式来绑定已定义的上游，它可以复用上游具有的健康检测、重试等功能。2、支持 `upstream` 和 `upstream_id` 的两种配置方式一起使用。
+:::tip 提示
+
+通过 `upstream_id` 方式来绑定已定义的上游，可以复用上游具有的健康检测、重试等功能。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:157]:
```suggestion
通过 `upstream_id` 方式来绑定已定义的上游，可以复用上游已存在的健康检测、重试等功能。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 8]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+支持同时使用 `upstream` 和 `upstream_id` 两种配置方式。
+
+:::
+
+## 测试插件
 
 ### 灰度发布
 
-缺少 `match` 规则部分，根据插件中 `weighted_upstreams` 配置的 `weight` 值做流量分流。将 `插件的 upstream` 与 `route 的 upstream` 按 3:2 的流量比例进行划分，其中 60% 的流量到达插件中的 `1981` 端口的 upstream， 40% 的流量到达 route 上默认 `1980` 端口的 upstream。
+灰度发布是逐步发布一个版本的过程，将越来越多的流量分给新版本，直到所有的流量都被引导至新版本。
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/traffic-split.md:171]:
```suggestion
灰度发布（又名金丝雀发布）是指在黑与白之间，能够平滑过渡的一种发布方式。 在其上可以进行 A/B 测试，即让一部分用户继续用产品特性 A，一部分用户开始用产品特性 B。如果用户对特性 B 没有什么反对意见，那么逐步扩大范围，把所有用户都迁移到特性 B 上面来。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 9]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
 
-traffic-split 插件使用户可以逐步引导各个上游之间的流量百分比。
+`traffic-split` 插件可以通过配置 `match` 和 `weighted_upstreams` 属性，从而动态地将部分流量引导至各种上游服务。该插件可应用于灰度发布和蓝绿发布的场景。
 
-注：由于加权循环算法（特别是在重置 wrr 状态时）的缺点，因此每个上游之间的比率可能不太准确。
+`match` 属性是用于引导流量的自定义规则，`weighted_upstreams` 属性则用于引导流量的上游服务。当一个请求被 `match` 属性匹配时，它将根据配置的 `weights` 属性被引导至上游服务。你也可以不使用 `match` 属性，只根据 `weighted_upstreams` 属性来引导所有流量。
+
+:::note 注意
+
+由于使用了加权循环算法（特别是在重置 `wrr` 状态时），因此在使用该插件时，可能会存在上游服务之间的流量比例不精准现象。
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:39]:
```suggestion
由于该插件使用了加权循环算法（特别是在重置 `wrr` 状态时），因此在使用该插件时，可能会存在上游服务之间的流量比例不精准现象。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 10]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+
+:::
 
 ## 属性
 
 |              参数名             | 类型          | 可选项 | 默认值 | 有效值 | 描述                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:45]:
```suggestion
|            名称             | 类型          | 必选项 | 默认值 | 有效值 | 描述                                                                                                                                                                                                                                                                                                                                                               |
```

Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:45]:
```suggestion
|            名称             | 类型          | 必选项 | 默认值 | 有效值 | 描述                                                                                                                                                                                                                                                                                                                                                               |
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 11]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
 }'
 ```
 
->注：1、通过 `upstream_id` 方式来绑定已定义的上游，它可以复用上游具有的健康检测、重试等功能。2、支持 `upstream` 和 `upstream_id` 的两种配置方式一起使用。
+:::tip 提示
+
+通过 `upstream_id` 方式来绑定已定义的上游，可以复用上游已存在的健康检测、重试等功能。
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:157]:
```suggestion
通过 `upstream_id` 方式来绑定已定义的上游，可以复用上游已存在的健康检查、重试等功能。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 12]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+
+:::
+
+:::note 注意
 
-## 示例
+支持同时使用 `upstream` 和 `upstream_id` 两种配置方式。
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:163]:
```suggestion
`weighted_upstreams` 属性支持同时使用 `upstream` 和 `upstream_id` 两种配置方式。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 13]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+
+:::
+
+:::info 重要
 
-## 如何启用
+在 `match` 属性中，变量中的表达式以 AND 方式关联，多个变量以 OR 方式关联。
 
-创建一个路由并启用 `traffic-split` 插件，在配置插件上游信息时，有以下两种方式：
+如果只配置了 `weight` 属性，那么它就对应于在 Route 或 Service 上配置的上游服务的权重。
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:72]: 如果你仅配置了 `weight` 属性，那么它将会使用该 Route 或 Service 中的上游服务的权重。

File: docs/zh/latest/plugins/traffic-split.md [hunk 14]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+支持同时使用 `upstream` 和 `upstream_id` 两种配置方式。
+
+:::
+
+## 测试插件
 
 ### 灰度发布
 
-缺少 `match` 规则部分，根据插件中 `weighted_upstreams` 配置的 `weight` 值做流量分流。将 `插件的 upstream` 与 `route 的 upstream` 按 3:2 的流量比例进行划分，其中 60% 的流量到达插件中的 `1981` 端口的 upstream， 40% 的流量到达 route 上默认 `1980` 端口的 upstream。
+灰度发布（又名金丝雀发布）是指在黑与白之间，能够平滑过渡的一种发布方式。 在其上可以进行 A/B 测试，即让一部分用户继续用产品特性 A，一部分用户开始用产品特性 B。如果用户对特性 B 没有什么反对意见，那么逐步扩大范围，把所有用户都迁移到特性 B 上面来。
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:171]:
```suggestion
灰度发布（又名金丝雀发布）是指在已经上线与未上线服务之间，能够平滑过渡的一种发布方式。 在其上可以进行 A/B 测试，即让一部分用户继续用产品特性 A，一部分用户开始用产品特性 B。如果用户对特性 B 没有什么反对意见，那么逐步扩大范围，把所有用户都迁移到特性 B 上面来。
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 15]
```diff
@@ -423,75 +464,74 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
 HTTP/1.1 200 OK
 Content-Type: text/html; charset=utf-8
-......
-
+...
 hello 1980
 ```
 
-在请求 5 次后，3 次命中 `1981` 端口的服务，2 次命中 `1980` 端口的服务。
+2. 如果第二个 `vars` 的表达式匹配失败（如缺少 `name2` 请求参数），`match` 规则校验通过后，效果将会与上一种相同。即有 60% 的请求被引导至插件 `1981` 端口的上游服务，40% 的请求命中到路由的 `1980` 端口的上游服务：
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:492]:
```suggestion
2. 如果第二个 `vars` 的表达式匹配失败（例如缺少 `name2` 请求参数），`match` 规则校验通过后，效果将会与上一种相同。即有 60% 的请求被引导至插件 `1981` 端口的上游服务，40% 的请求命中到路由的 `1980` 端口的上游服务：
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 16]
```diff
@@ -551,35 +591,45 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
 ```
 
 ## 禁用插件
 
-当你想去掉 traffic-split 插件的时候，很简单，在插件的配置中把对应的 json 配置删除即可，无须重启服务，即刻生效：
+当你需要禁用该插件时，可以通过以下命令删除相应的 JSON 配置，APISIX 将会自动重新加载相关配置，无需重启服务：
 
 ```shell
-$ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+$ curl http://127.0.0.1:9180/apisix/admin/routes/1 \
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:631]:
```suggestion
curl http://127.0.0.1:9180/apisix/admin/routes/1 \
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 17]
```diff
@@ -23,53 +30,56 @@ title: traffic-split
...
+| upstream.pass_host             | enum   | 否   | "pass"   | ["pass", "node", "rewrite"]  | 当请求被转发到上游时配置 `host`。`pass` 代表将客户端的 `host` 透明传输给上游；`node` 代表使用 `upstream` Node 中配置的 `host`； `rewrite` 代表使用配置项 `upstream_host` 的值。                                                                                                                                                                                                                                                                |
+| upstream.name                  | string | 否   |        |  | 标识上游服务名称、使⽤场景等。                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:58]:
```suggestion
| upstream.name                  | string | 否   |        |  | 标识上游服务名称、使用场景等。                                                                                                                                                                                                                                                                                                                                                  |
```

File: docs/zh/latest/plugins/traffic-split.md [hunk 18]
```diff
@@ -141,16 +152,29 @@ curl http://127.0.0.1:9180/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f13
...
+:::
+
+## 测试插件
 
 ### 灰度发布
 
-缺少 `match` 规则部分，根据插件中 `weighted_upstreams` 配置的 `weight` 值做流量分流。将 `插件的 upstream` 与 `route 的 upstream` 按 3:2 的流量比例进行划分，其中 60% 的流量到达插件中的 `1981` 端口的 upstream， 40% 的流量到达 route 上默认 `1980` 端口的 upstream。
+灰度发布（又名金丝雀发布）是指在黑与白之间，能够平滑过渡的一种发布方式。 在其上可以进行 A/B 测试，即让一部分用户继续用产品特性 A，一部分用户开始用产品特性 B。如果用户对特性 B 没有什么反对意见，那么逐步扩大范围，把所有用户都迁移到特性 B 上面来。
+
+以下示例展示了如何通过配置 `weighted_upstreams` 的 `weight` 属性来实现流量分流。按 3:2 的权重流量比例进行划分，其中 60% 的流量到达运行在 `1981` 端口上的上游服务，40% 的流量到达运行在`1980` 端口上的上游服务：
```
Comment (hf400159) [inline docs/zh/latest/plugins/traffic-split.md:173]:
```suggestion
以下示例展示了如何通过配置 `weighted_upstreams` 的 `weight` 属性来实现流量分流。按 3:2 的权重流量比例进行划分，其中 60% 的流量到达运行在 `1981` 端口上的上游服务，40% 的流量到达运行在 `1980` 端口上的上游服务：
```

**Review by SylviaBABY — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**